bot/cogs/manage.py

```diff
@@ -16,6 +21,24 @@ async def manage(self, ctx):
         await ctx.send("No subcommand was found!")
         await ctx.send_help(self.manage)
 
+    @manage.command(help="Toggle the Admin role for you or a mentioned user")
```

Comment:
I think you should use a try-except clause here

Reply:
Where exactly?